packages/dds/tree/src/simple-tree/types.ts

```diff
@@ -59,7 +122,7 @@ export type Unhydrated<T> = T;
  *
  * @sealed @public
  */
-export interface TreeChangeEvents {
+export interface TreeChangeEvents extends TreeChangeEventsBase {
```

Comment:
`TreeChangeEvents` stlil handles undifferentiated `TreeNode`, providing no payload for changedProperties. Not sure if should have it for everything, but I think trying to put it there led me to other errors.